rust/template/differential_datalog/profile.rs

```diff
@@ -74,11 +132,27 @@ impl Profile {
     }
 
     fn handle_timely(&mut self, msg: &Vec<((Duration, usize, TimelyEvent), String)>) {
-        for ((_, _, event), ctx) in msg.iter() {
+        for ((ts, worker, event), ctx) in msg.iter() {
```

Comment:
Should this be controlled by some compile flag - in case it's expensive?

Reply:
Does not seem to be (although it may be workload dependent). Adding an argument to the Rust API is not a problem, but C and Java APIs will have to change as well. Leaving it as is for now.